runtime/lua/vim/treesitter/dev.lua

```diff
@@ -39,7 +39,7 @@ local TSTreeView = {}
 ---@param node TSNode Starting node to begin traversal |tsnode|
 ---@param depth integer Current recursion depth
 ---@param lang string Language of the tree currently being traversed
----@param injections table<integer,TSP.Node> Mapping of node ids to root nodes of injected language trees (see
+---@param injections table<string,table> Mapping of node ids to root nodes of injected language trees (see
```

Comment:
Is there not a better type we can use instead of table?

Reply:
`{ lang: string, root: TSNode }` represents the exact content stored by injections table.

Reply:
So basically a subset of `TSP.Node`? If so we can add a class for this.

Reply:
I added `TSP.Injection` in f2d10102b6ed825c251f3cc97305767b7b4abd7f.